mypy/build.py

```diff
@@ -1282,10 +1307,11 @@ def write_cache(id: str, path: str, tree: MypyFile,
         return interface_hash, None
 
     # Write data cache file, if applicable
+    # Note that for Bazel we don't record the data file's mtime.
     if old_interface_hash == interface_hash:
         # If the interface is unchanged, the cached data is guaranteed
         # to be equivalent, and we only need to update the metadata.
-        data_mtime = getmtime(data_json)
+        data_mtime = 0 if bazel else getmtime(data_json)
```

Comment:
Maybe put "0 if bazel else getmtime(path)" into a one-line function? (it appears in a lot of places and causes a bit of cognitive load each time).